Multi shop view support (#100)
shopgate/cart-integration-shopware: README.md

You can find more information in our [support center](https://support.shopgate.com/hc/en-us/articles/202798446-Connecting-to-Shopware) and our [developer documentation](https://docs.shopgate.com/).

> ## Shop multi views and translations

Reviewer: ```suggestion
## Shop Multi Views and Translations
```

> We need distinguish url of plugin location to allow Shopware load correct shop view data, locale and configuration

Reviewer: ```suggestion
We need distinguishable URLs for the plugin location to enable Shopware loading the correct shop view data, locale and configuration.
```

> Shop settings generally should be
> 
> - `virtual Url` field point to (example)
>     - /shop/de for DE shop
>     - /shop/en for EN shop
> 
> Plugin access uri should be accordingly:
> - /shop/de/shopgate/plugin for DE shop view and translations
> - /shop/en/shopgate/plugin for EN shop view and translations

Reviewer: ```suggestion
Example:

If the shop is set up to have its virtual URL point to
- `/shop/de` for the German shop
- `/shop/en` for the English shop

then access to the plugin should be
- `/shop/de/shopgate/plugin` to get contents (catalog etc.) in German
- `/shop/en/shopgate/plugin` to get contents (catalog etc.) in English
```
